### PR TITLE
dev

### PR DESCRIPTION
**Pull request by robertavram**

### CODE REVIEW

File: src/etools_datamart/apps/mart/data/models/partners_interventionbudget.py [hunk 1]
```diff
@@ -1,32 +1,161 @@
...
+                    queryset=ReportsResult.objects.only("id", "name", "wbs"),
+                ),
+            )
+        )
 
     def process_country(self):
         batch_size = settings.RESULTSET_BATCH_SIZE
         logger.debug(f"Batch size:{batch_size}")
-
-        qs = self.get_queryset().exclude(intervention__isnull=True)
```
Comment (robertavram) [inline src/etools_datamart/apps/mart/data/models/partners_interventionbudget.py:29]: add this back in.

Comment (hbarisik) [reply src/etools_datamart/apps/mart/data/models/partners_interventionbudget.py:29]: PartnersInterventionbudget.objects.exclude(intervention__isnull=True)  is already there.

File: src/etools_datamart/apps/etl/tasks.py [hunk 2]
```diff
@@ -20,3 +20,10 @@ def healthcheck():
 # @app.on_after_configure.connect
 # def setup_periodic_tasks(sender, **kwargs):
 #     sender.add_periodic_task(60.0, healthcheck.s(), name='healthcheck')
+
+
+@app.task()
```
[the remark this answers is not in the document]
Comment (hbarisik) [reply src/etools_datamart/apps/etl/tasks.py:25]: Removed in PR for [ch38092] - still to be merged to develop

**Review by emaciupe — APPROVED** [review]
Looks good!

**Pull request merged**